app/(main)/store/home/_components/business-status/BusinessStatus.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+
+import ToggleSwitch from '../../../../../../app/_components/toggle-switch/ToggleSwitch';
+import { useState } from 'react';
+
+const BusinessStatus = () => {
+  const [isOff, setIsOff] = useState(true);
+
+  const getBusinessStatus = (toggle: boolean) => {
+    setIsOff(toggle);
+    /** @todo 영업 상태 변경 api 연결 추가 */
+  };
+
+  return (
+    <div className="flex h-11 w-full items-center justify-between rounded bg-white shadow-sm">
+      <div className="flex items-center gap-x-1.5 pl-3">
+        <div
+          className={`h-4 w-4 rounded-full ${!isOff ? 'bg-cyan' : 'bg-red'}`}
```

Comment:
부정문을 쓰기 보다는 순서만 반대로 해도 이해하기 편할 것 같아요!

```suggestion
          className={`h-4 w-4 rounded-full ${isOff ? 'bg-red' : 'bg-cyan'}`}
```

Reply:
수정하겠습니다.